accounts/src/main/java/eu/kevin/accounts/accountsession/AccountSession.kt

```diff
@@ -148,7 +148,8 @@ internal class AccountSession(
                     configuration.countryFilter,
                     configuration.bankFilter,
                     sessionData.selectedBank?.id,
-                    configuration.state
+                    configuration.state,
+                    false
```

Comment:
great example where having a named parameter would make sense, because it's unclear what's false.
`keepAccountLinkingNotSupportedBanks = false`